spec/kintsugi_apply_change_to_project_spec.rb

```diff
@@ -984,6 +985,33 @@
           configuration.build_settings["HEADER_SEARCH_PATHS"] = "baz"
         end
 
+        expected_project = create_copy_of_project(base_project.path, "expected")
+        expected_project.targets[0].build_configurations.each do |configuration|
+          configuration.build_settings["HEADER_SEARCH_PATHS"] = %w[bar baz]
+        end
```

Comment:
nit: Move to be just before being used, i.e. before `expect`

Reply:
I can't because we're applying the changes to `base` and I need to make the copy before that